C/common/config_category.cpp

```diff
@@ -1385,6 +1390,28 @@ ConfigCategory::CategoryItem::CategoryItem(const string& name,
 		}
 	}
 
+	if (item.HasMember("schema"))
+	{
+		Logger::getLogger()->debug("item['schema'].IsString()=%s, item['schema'].IsObject()=%s",
+										item["schema"].IsString()?"true":"false",
+										item["schema"].IsObject()?"true":"false");
+
+		rapidjson::StringBuffer strbuf;
+		rapidjson::Writer<rapidjson::StringBuffer> writer(strbuf);
+		item["schema"].Accept(writer);
+		m_jsonSchema = item["schema"].IsObject() ?
+			  // use current string
+			  strbuf.GetString() :
```

Comment:
We should add some tests around these conditions based samples. Plus, validating how JSON Schema can be a non-object?